EIPS/eip-2381.md

```diff
@@ -0,0 +1,143 @@
+---
+eip: 2381
+title: ENS support for EIP-721 TokenID
+author: Oisin Kyne (@OisinKyne)
+discussions-to: https://github.com/ethereum/EIPs/issues/2381
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-11-15
+requires: 137, 165, 721
+---
+
+## Simple Summary
+
+This EIP makes [EIP-721](https://eips.ethereum.org/EIPS/eip-721) Non-Fungible Tokens addressable by the Ethereum Name Service ([EIP-137](https://eips.ethereum.org/EIPS/eip-137)) by adding a new resolver profile, `uint256 tokenID()`. This allows the Ethereum Name Service to address a single non-fungible token within an EIP-721 token contract of many tokens.
```

Comment:
This is still longer than we would like.  Think email subject line, or Git commit message header, or the last column in a table.

Reply:
Cut it down significantly and moved things to motivation